spec/assessments/countWordAssessmentSpec.js

```diff
@@ -36,7 +37,8 @@ var repeatString = function(string, times) {
 describe( "A word count assessment", function(){
 	it( "assesses a single word", function(){
 		var mockPaper = new Paper( "sample" );
-		var assessment = wordCountAssessment( mockPaper, i18n );
+		var researcher = new Researcher( mockPaper );
```

Comment:
Currently with this test we are testing both the word count research and the word count assessment. We could only test the assessment by mocking the researcher:

```
var researcher = {
    getResearch: function() { return 1; }
};
```